View by host/by service (#2711)
centreon/centreon-documentation: versioned_docs/version-23.10/alerts-notifications/resources-status.md

![image](../assets/alerts/resources-status/resources-status-listing-order.gif)

### View by host or by service

Use the **Display view** buttons above the list of resources to define how the list will be displayed. 

> - **All**: This is the classic Resources Status view, with all resources gathered in the same view (hosts, services, metaservices, Anomaly Detection services)

Author: ```suggestion
- **All**: This view gathers all types of resources in the same list (hosts, services, metaservices, Anomaly Detection services).
```

> - **View by host**: Displays all hosts. By default, for each host, a summary of the status of all services is displayed in the **Services** column.  Click the arrow to the left of the services to expand the zone and display all the services attached to this host. Meta services are not included in this view.

Author: ```suggestion
- **View by host**: Displays all hosts. By default, for each host, a summary of the status of all services is displayed in the **Services** column. Click the arrow to the left of the summary to expand the zone and display all the services attached to this host. Meta services are not included in this view.
```
